classes/log/store.php

```diff
@@ -167,6 +168,25 @@ public function get_max_batch_size() {
         return $this->get_config('maxbatchsize', 100);
     }
 
+    /**
+     * Take event data from the emit_task
+     * and add to the sent log if it doesn't exist already.
+     *
+     * @param array $events raw event data
+     * @return array
+     */
+    private function add_event_to_sent_log($event) {
+        global $DB;
+        $row = $DB->get_record('logstore_xapi_sent_log', ['logstorestandardlogid' => $event->logstorestandardlogid]);
+        if (empty($row)) {
+            $newrow = new stdClass();
```

Comment:
could this be `new php_obj();` ?

Reply:
Hi Stephen,
Yes this could be php_obj();
The original code uses:
use \stdClass as php_obj;
Namespace aliases were added in PHP 5.6 and it is used in the extends clause in this class declaration but I don't think it adds clarity, so I used stdClass. Would you prefer me to use php_obj?

Reply:
I understand your point, Michael but I don't think we should use more ways the same element so I agree with Stephen.
Otherwise we should delete the alias and update everywhere which is bigger task than we use this alias.

Reply:
I will use the alias.

Reply:
I have removed this code and added i to the appropriate cron sub-task.